presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
```

Comment:
yeah.. it is much shorter than `prestoConfigUserAlice` and means the same. To me it is more readable. What do you think about that?